apis/kueue/v1alpha1/workload_webhook_test.go

```diff
@@ -0,0 +1,56 @@
+package v1alpha1
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe("Defaulting webhook", func() {
+	Context("When creating Workload", func() {
+		It("Should set default value ", func() {
+			By("Creating a new Workload")
+			workload := &Workload{
```

Comment:
use the wrappers pkg; see this example https://github.com/kubernetes-sigs/kueue/blob/d98cedd7ccfdb9234a6ebe7a6a0c0de83085585d/test/integration/controller/core/workload_controller_test.go#L61